terra-jupyter-aou/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN find $JUPYTER_HOME/scripts -name '*.sh' -type f | xargs chmod +x \
     && (cd $X && pip3 download hail==$HAIL_VERSION --no-dependencies && \
         unzip hail*.whl && \
         grep 'Requires-Dist: ' hail*dist-info/METADATA | sed 's/Requires-Dist: //' | sed 's/ (//' | sed 's/)//' | grep -v 'pyspark' >$requirements_file && \
-        pip install -r $requirements_file) \
+        pip install --verbose -r $requirements_file) \
```

Comment:
did you add verbose for your debugging only or you wanted it released?

Reply:
Marcus added that and I never got the answer for why. I will remove.

Reply:
yes my bad.  it was for debugging on my laptop